__test__/Block.spec.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+configure({ adapter: new Adapter() });
+
+import { factory as blockFactory } from '../src/block';
+import { defaultConfig } from '../src/blockConfig';
+
+describe('The Starter Block', () => {
+    const StarterBlock = blockFactory(React);
+
+    describe('when there is no custom data', () => {
+        it('should render the block with the default content', () => {
+            const wrapper = shallow(<StarterBlock />);
+            expect(wrapper.text()).toBe(defaultConfig.text);
+            expect(wrapper).toMatchSnapshot();
+        });
+    });
+
+    describe('when given custom data', () => {
+        it('should render the block using the custom data', () => {
```

Comment:
Avoid using the term "render" in your test descriptions since you're not using the enzyme `render` method.

Reply:
See above: https://github.com/volusion/element-BlockStarter/pull/8#discussion_r303578314